### PR TITLE
Identify poison messages scenarios

### PR DESCRIPTION
**Pull request by mauroservienti**

### CODE REVIEW

**Review by mikeminutillo — APPROVED** [review]

File: src/NServiceBus.Transport.SQS.TransportTests/Sending_poison_messages.cs [hunk 1]
```diff
@@ -33,29 +34,40 @@ public async Task Should_move_to_error_queue(TransportTransactionMode transactio
...
             await CheckErrorQueue(ErrorQueueName, cancellationTokenSource.Token);
 
             Assert.False(onErrorCalled, "Poison message should not invoke onError");
             Assert.False(onMessageCalled, "Poison message should not invoke onMessage");
         }
 
-        string PoisonMessageBody = "this is a poison message that won't deserialize to valid json";
-
-        async Task SendPoisonMessage(string inputQueueName)
+        // string PoisonMessageBody = "this is a poison message that won't deserialize to valid json";
```
Comment (mauroservienti) [inline src/NServiceBus.Transport.SQS.TransportTests/Sending_poison_messages.cs:62]: Delete this

Comment (mauroservienti) [reply src/NServiceBus.Transport.SQS.TransportTests/Sending_poison_messages.cs:62]:
```suggestion
```

**Comment by mauroservienti** [comment]
I went through the input queue pump code and could not identify any other poison message scenario.

**Comment by mauroservienti** [comment]
Flaky test. Merging.

**Pull request merged**